src/enums/enum.ts

```diff
@@ -1,10 +1,10 @@
 export enum TeamEvents {
```

Comment:
Do we use anywhere this enum? If not I would recommend to remove it and add when it will need needed.

Reply:
Sebastian wrote: 
use 'export' before enum declaration
We want to use enum soon ;-)